### DISCUSSION TITLE
Git hub Education Student approved but on waiting period. How long does it take?

### DISCUSSION THREAD

**Discussion by darshineeC** [community/community#130101] [GitHub Education]

**Comment by PipoAT** [comment]
For student benefits it normally takes up to 72 hours after approval to kick in, however, it may be longer (by an undetermined amount) if it is a busy period for requests to be coming in at the same time (i.e. start of the new academic semester).

**Reply by aswinpradeepc** [reply]
Hi, but it has been almost 2 months and I haven't got access to even GitHub's free student resources like copilot. What can be done about this ?

**Comment by darshineeC** [comment]
I have received the email for welcome on global campus but no access to github copilot

**Comment by Mayank-cyber-cell** [comment]
Hi! Usually, GitHub Student Developer Pack benefits (including Copilot Pro) are activated within 72 hours after approval, but during busy periods it can sometimes take longer.

Since you already received the Global Campus welcome email, your approval likely went through successfully. If Copilot or other student benefits are still unavailable after several days/weeks, here are a few things you can try:

• Make sure your GitHub account is showing as verified student status in your settings.
• Try signing out and signing back into GitHub.
• Check whether GitHub Copilot needs to be manually activated from the benefits page.
• Visit the Student Developer Pack dashboard and confirm the benefits are marked as available.
• If it has been more than a couple of weeks, contact GitHub Education Support directly with screenshots of your approval email and account status.

Many users have reported delayed activation during high-traffic periods, so you’re definitely not alone. Hope this gets resolved soon!
